### ISSUE TITLE
[FEATURE] Track test failures against past test runs

### ISSUE THREAD

**Issue by forgxyz** [re-data/re-data#268]
**Tell us about the problem you're trying to solve**

Disclosure - we have not yet implemented re_data, but are considering it. I love that the test failures are easy to spot, such as with [this example](https://docs.getre.io/ui-master/#/tests/source_not_null_toy_shop_sources_toy_shop_customers_age).

I would love for NEW failures to be highlighted. So, if on all previous runs Test X passed, but it is now failing, that information is more important to me than the regular failed tests, as we are presumably working on a fix for those.

Similarly, it would be nice to be shown when an event change happens the other way! I.e. a consistent failure now passes!

**Describe the solution you’d like**

Basically, linking tests and their error status across runs. This would be especially powerful if we could visualize the results over time. So, instead of clicking through past runs in the dropdown (which is great), seeing a timeline of those runs. This would let us pinpoint when a test failed and how long it took to deliver a fix!

**Describe the alternative you’ve considered or used**

Nothing at the moment, I would like to propose implementing re_data to my team as the error status page is already an improvement over manually checking failures.
